src/backend/src/services/change-requests.services.ts

```diff
@@ -614,87 +613,93 @@ export default class ChangeRequestsService {
       }
     });
 
-    if (wbsProposedChanges) {
-      if (projectProposedChanges && workPackageProposedChanges) {
-        throw new HttpException(400, "Change Request can't be on both a project and a work package");
-      } else if (!projectProposedChanges && !workPackageProposedChanges) {
-        throw new HttpException(
-          400,
-          'Change Request with proposed changes must have either project or work package proposed changes'
-        );
-      } else {
-        const { name, status, projectLeadId, projectManagerId, links } = wbsProposedChanges;
+    let name: string = '';
+    let status: WbsElementStatus = WbsElementStatus.Active;
+    let projectLeadId: number = 0;
+    let projectManagerId: number = 0;
+    let links: { url: string; linkTypeName: string }[] = [];
 
-        if (projectLeadId) {
-          const projectLead = await prisma.user.findUnique({ where: { userId: projectLeadId } });
-          if (!projectLead) throw new NotFoundException('User', projectLeadId);
-        }
+    if (projectProposedChanges && workPackageProposedChanges) {
+      throw new HttpException(400, "Change Request can't be on both a project and a work package");
+    } else if (!projectProposedChanges && !workPackageProposedChanges) {
+      throw new HttpException(
+        400,
+        'Change Request with proposed changes must have either project or work package proposed changes'
+      );
+    } else if (projectProposedChanges) {
+      ({ name, status, projectLeadId, projectManagerId, links } = projectProposedChanges);
```

Comment:
this syntax is wrong, try `const {name, your other stuff} = projectProposedChanges; ` then you shouldn't need to declare the variables before this line (lines 616-620)